dojson/utils.py

```diff
@@ -30,7 +30,7 @@ def int_with_default(value, default):
 
 
 def ignore_value(f):
-    """Remove key for None value.
+    """Remove key for None or empty value value.
```

Comment:
Is it? I remember cataloguers are struggling with the difference of an empty value and a non existing field. Also traditionally you can't search for empty strings.
@aw-bib is there a value in MARC to preserve empty values?